setup.cfg

```diff
@@ -29,6 +29,7 @@ exclude =
     doc/build/*.py,
     doc/temp/*.py,
     .eggs/*.py,
+    pandas/_vendored/*
```

Comment:
Is this still required to ignore?

Reply:
either way i guess

Reply:
Let's remove if not necessary; less things to ponder is always better